pkg/util/validation/validation.go

```diff
@@ -109,3 +124,13 @@ func EmptyOverrides(overriders policyv1alpha1.Overriders) bool {
 	}
 	return true
 }
+
+// ValidateLabels validates that a set of labels are correctly defined.
+func ValidateLabels(overrider policyv1alpha1.LabelAnnotationOverrider, overrideRuleIndex, labelsOverriderIndex int) error {
+	return metav1validation.ValidateLabels(overrider.Value, field.NewPath("spec", "overrideRules").Index(overrideRuleIndex).Child("overriders", "labelsOverrider").Index(labelsOverriderIndex).Child("value")).ToAggregate()
```

Comment:
Can the `fldPath` field be set layer by layer to facilitate subsequent expansion and check other fields?

Reply:
Done